packages/ui/src/elements/WhereBuilder/Condition/validOperators.ts

```diff
@@ -0,0 +1,20 @@
+import type { Operator } from 'payload'
+
+export const operatorValueTypes: Record<Operator, 'any' | 'boolean' | 'object' | 'string'> = {
+  all: 'any',
+  contains: 'string',
+  equals: 'any',
+  exists: 'boolean',
+  greater_than: 'object',
+  greater_than_equal: 'object',
+  in: 'any',
+  intersects: 'any',
+  less_than: 'object',
+  less_than_equal: 'object',
+  like: 'string',
+  near: 'any',
+  not_equals: 'any',
+  not_in: 'any',
+  not_like: 'string',
+  within: 'any',
```

Comment:
Can we add some comments in here with your findings about field types, i.e. the whole dates vs numbers for things like `less_than` etc. The comments can just be above each line, might make sense to use comment blocks instead of single line comments but thats up to you. I just think it will be helpful for future usage of this file.

Reply:
Yes absolutely - I've added comments for the operators which behave differently depending on the field type. I also added a comment for `exists`, which should be a boolean but is instead the string of `true` or `false`.